src/test/scala/cognite/spark/v1/SparkTest.scala

```diff
@@ -246,11 +246,11 @@ trait SparkTest {
 
   val updateAndUpsert: TableFor1[String] = Table(heading = "mode", "upsert", "update")
 
-  def getDefaultConfig(auth: CdfSparkAuth, projectName: String): RelationConfig =
+  def getDefaultConfig(auth: CdfSparkAuth, projectName: String, cluster: String = "api", applicationName: Option[String] = Some("SparkDatasourceTestApp")): RelationConfig =
```

Comment:
nit: maybe it shouldn't be called "default config" anymore with new customization options

Reply:
Vast majority is still with default, just with some more override, also they have default value.  I think the spirit is still there ^^